### ISSUE TITLE
`\boldsymbol{\mathrm{b}}` in boldface

### ISSUE THREAD

**Issue by hbghlyj** [ronkok/Temml#45]
In [LaTeX](https://texlive.net/run?%5Cdocumentclass%7Barticle%7D%0A%5Cusepackage%7Bamsmath%7D%0A%5Cbegin%7Bdocument%7D%0A%24%5Cboldsymbol%7B%5Cmathrm%7Bb%7D%7D%24%0A%5Cend%7Bdocument%7D)
```tex
\documentclass{article}
\usepackage{amsmath}
\begin{document}
$\boldsymbol{\mathrm{b}}$
\end{document}
```
outputs the boldface **b**

But in Temml `\boldsymbol{\mathrm{b}}` outputs the regular b
![image](https://github.com/ronkok/Temml/assets/53823634/f8fa6653-af5b-4c56-aaad-85c1d2e52440)

**Comment by ronkok** [comment]
I knew that `\boldsymbol` behaved differently than other font functions. I guess there are more nuances than I was aware of. I'll look into it.

**Comment by ronkok** [comment]
Fixed by release v0.10.20. Thanks for raising this issue.
